File: DWES/calculadora.py
class Calculadora: 
    
    def __init__(self,num1,num2):
        self.num1 = num1
        self.num2 = num2
    
    def sumar(self):
        return self.num1 + self.num2 
    
    def restar(self):
        return self.num1 - self.num2
    
    def multiplicar(self):
        return self.num1 * self.num2
    
    def dividir(self):
        if num2 != 0:
            return self.num1 / self.num2  
        else:
            return ("No se puede dividir por cero")
        
num1 = int(input("Introduce el primer numero: "))
num2 = int(input("Introduce el segundo numero: "))


suma = Calculadora(num1,num2)
#resta = Calculadora(num1,num2)
#multiplicacion = Calculadora(num1,num2)
#division = Calculadora(num1,num2)

#print(suma.sumar())
#print(resta.restar())
#print(multiplicacion.multiplicar())
#print(division.dividir())

File: DWEC/JavaScript/calculadora.js

let operacion = parseInt(prompt("¿Qué operación quieres realizar?\n 1-suma,\n 2-resta,\n 3-multiplica,\n 4-divide"));
let resultado;
let num1;
let num2;

do {
    num1 = parseFloat(prompt("Dime el primer número: "));
    if (isNaN(num1)) {
        alert("Eso no es un número");
    }
} while (isNaN(num1));

do {
    num2 = parseFloat(prompt("Dime el segundo número: "));
    if (isNaN(num2)) {
        alert("Eso no es un número");
    }
} while (isNaN(num2));

switch (operacion) {
    case 1:
        resultado = num1 + num2;
        alert(`El resultado de la suma es: ${resultado}`);
        break;
    case 2:
        resultado = num1 - num2;
        alert(`El resultado de la resta es: ${resultado}`);
        break;
    case 3:
        resultado = num1 * num2;
        alert(`El resultado de la multiplicación es: ${resultado}`);
    case 4:
        if (num2 === 0) {
            alert("No se puede dividir entre 0");
        } else {
            resultado = num1 / num2;
            alert(`El resultado de la división es: ${resultado}`);
        }
        break;
    default:
        alert("Operación no válida");
        break;
}